### PR TITLE
RAW coplist support in scrollbuffer and tilebuffer

### PR DESCRIPTION
**Pull request by timfel**
It's not finished, at the very least it's missing a showcase that uses it. But I wanted to open it to get feedback already.

### CODE REVIEW

File: src/ace/managers/viewport/scrollbuffer.c [hunk 1]
```diff
@@ -75,22 +75,42 @@ tScrollBufferManager *scrollBufferCreate(void *pTags, ...) {
 	// Create copperlist entries
 	pCopList = pVPort->pView->pCopList;
 	if(pCopList->ubMode == COPPER_MODE_BLOCK) {
-		pManager->pStartBlock = copBlockCreate(
+		pManager->copStart.pStartBlock = copBlockCreate(
```
Comment (tehKaiN) [inline src/ace/managers/viewport/scrollbuffer.c:78]: those can be anonymous unions to skip the copStart/copBreak stuff, or if not, union contents can have shorter names to not duplicate the "start/break" and "copper" info from union name.

Something like `pManager->copStart.pBlock` and `pManager->copStart.uwOffset`

File: include/ace/managers/viewport/tilebuffer.h [hunk 2]
```diff
@@ -170,11 +170,6 @@ void tileBufferDestroy(tTileBufferManager *pManager);
  */
 void tileBufferProcess(tTileBufferManager *pManager);
 
-void tileBufferReset(
-	tTileBufferManager *pManager, UWORD uwTileX, UWORD uwTileY,
-	UBYTE ubBitmapFlags, UBYTE isDblBuf
-);
-
```
Comment (tehKaiN) [inline include/ace/managers/viewport/tilebuffer.h:177]: why removed tileBufferReset fn?

**Review by tehKaiN — CHANGES_REQUESTED** [review]
Looks good but I'm a bit confused why you've removed tileBufferReset() from tilebuffer.h header - this is completely valid non-static fn to call when e.g. reloading levels.

Or am I'm missing something due to github diff view? ;)

File: include/ace/managers/viewport/tilebuffer.h (hunk 2 above)
Comment (timfel) [reply include/ace/managers/viewport/tilebuffer.h:177]: you're right, I moved it to `tilebuffer.c` for no good reason.

File: src/ace/managers/viewport/scrollbuffer.c (hunk 1 above)
Comment (timfel) [reply src/ace/managers/viewport/scrollbuffer.c]: Made it anonymous

File: include/ace/managers/viewport/tilebuffer.h (hunk 2 above)
Comment (timfel) [reply include/ace/managers/viewport/tilebuffer.h]: moved back to header

**Review by tehKaiN — APPROVED** [review]

**Pull request merged**